mod-users/src/test/java/org/folio/ModUsersTests.java

```diff
@@ -0,0 +1,32 @@
+package org.folio;
+
+import org.folio.test.TestBase;
+import org.folio.test.config.TestModuleConfiguration;
+import org.folio.test.services.TestIntegrationService;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ModUsersTests extends TestBase {
+
+  private static final String TEST_BASE_PATH = "classpath:domain/mod-users/features/";
```

Comment:
This is only used in one place, why is it a class level constant?

Reply:
i am not exactly sure about it but i guess because of TestBase class constructor, since in all other projects it is same so i have kept as it is.

Reply:
I don't understand. 

How could a (non-overriding) variable defined in the sub-class have any impact on what is being done in the super-class?